hoodie-spark/src/test/java/HoodieJavaApp.java

```diff
@@ -42,7 +42,7 @@
 /**
  * Sample program that writes & reads hoodie datasets via the Spark datasource
  */
-public class HoodieJavaApp {
+public class HoodieJavaApp extends AbstractCommandConfig {
```

Comment:
Instead of this, can  you define an inner Config class and have it extend AbstractCommandConfig